src/server/lib/serverSideLogger.ts

```diff
@@ -25,6 +26,22 @@ class ServerSideLogger extends BaseLogger {
 		error?: any,
 		extraFields?: ExtraLogFields,
 	) {
+		const context = requestContext.getStore();
+
+		const extraFieldsWithContext = {
+			...extraFields,
+			...(context
+				? {
+						in_request_context: true,
+						request_id:
+							context.request.headers?.['x-request-id'] ?? 'no-request-id',
+						ip: context.request.ip ?? 'no-ip',
```

Comment:
My personal opinion would be for it to be `undefined` or an empty string if the request id or ip doesn't exist. But this is just preference.

Reply:
I'm ambivalent! Lets go with the less code way!